src/etc/inc/ipsec.inc

```diff
@@ -2132,6 +2140,18 @@ function ipsec_setup_tunnels() {
 			$conn['tfc_padding'] = (isset($ph1ent['tfc_bytes']) && is_numericint($ph1ent['tfc_bytes'])) ? $ph1ent['tfc_bytes'] : "mtu";
 		}
 
+		/* Custom Ports */
+		if (isset($ph1ent['ikeport'])) {
+			/* we want to initiate an ike connection without encaps, so we don't set local_port */
+			$conn['remote_port'] = $ph1ent['ikeport'];
+		}
+		else if (isset($ph1ent['nattport'])) {
```

Comment:
} and else must be in the same line

Reply:
Also, use `elseif`, not `else if`: https://docs.netgate.com/pfsense/en/latest/development/developer-style-guide.html#general-rules